packages/common-common/src/redisCache.ts

```diff
@@ -136,10 +136,14 @@ export class RedisCache {
     }
     const finalKey = RedisCache.getNamespaceKey(namespace, key);
 
+    if (typeof value !== 'string') {
+      value = JSON.stringify(value);
+    }
+
```

Comment:
@NakulManchanda Does this make sense to add here? I was able to send some objects  to this point, so this helped.

Reply:
- if possible lets always send strings for now, may be some boundary condition which can arise, lets keep cache logic simple itself

- but make total sense if its being repeated again n again

JSON.stringify do same much safer than JSON.parse for sure -> it can stringify non-json objects too

Reply:
We need to keep this I think. I don't think I can intercept the RPC requests easily to make sure they send a string.